com.unity.netcode.gameobjects/Tests/Editor/Messaging/MessageRegistrationTests.cs

```diff
@@ -237,65 +251,16 @@ public void MessagesGetPrioritizedCorrectly()
             var provider = new OrderingMessageProvider();
             using var messagingSystem = new MessagingSystem(sender, null, provider);
 
-            // the 3 priority messages should appear first, in lexicographic order
+            // the 2 priority messages should appear first, in lexicographic order
             Assert.AreEqual(messagingSystem.MessageTypes[0], typeof(ConnectionApprovedMessage));
             Assert.AreEqual(messagingSystem.MessageTypes[1], typeof(ConnectionRequestMessage));
-            Assert.AreEqual(messagingSystem.MessageTypes[2], typeof(OrderingMessage));
 
             // the other should follow after
-            Assert.AreEqual(messagingSystem.MessageTypes[3], typeof(AAAEarlyLexicographicNetworkMessage));
-            Assert.AreEqual(messagingSystem.MessageTypes[4], typeof(zzzLateLexicographicNetworkMessage));
-
-            // there should not be any extras
-            Assert.AreEqual(messagingSystem.MessageHandlerCount, 5);
-
-            // reorder the zzz one to position 3
-            messagingSystem.ReorderMessage(3, XXHash.Hash32(typeof(zzzLateLexicographicNetworkMessage).FullName));
-
-            // the 3 priority messages should still appear first, in lexicographic order
-            Assert.AreEqual(messagingSystem.MessageTypes[0], typeof(ConnectionApprovedMessage));
-            Assert.AreEqual(messagingSystem.MessageTypes[1], typeof(ConnectionRequestMessage));
-            Assert.AreEqual(messagingSystem.MessageTypes[2], typeof(OrderingMessage));
-
-            // the other should follow after, but reordered
+            Assert.AreEqual(messagingSystem.MessageTypes[2], typeof(AAAEarlyLexicographicNetworkMessage));
```

Comment:
Wow, this update made this test much simpler!

Reply:
Mostly just because OrderingMessage was removed (its functionality was rolled into ConnectionApprovedMessage)